Sources/SwifterSwift/SwiftStdlib/OptionalExtensions.swift

```diff
@@ -111,6 +111,111 @@ public extension Optional where Wrapped: Collection {
 
 }
 
+// MARK: - Methods (RawRepresentable, RawValue: Equatable)
+public extension Optional where Wrapped: RawRepresentable, Wrapped.RawValue: Equatable {
+
+    // swiftlint:disable missing_swifterswift_prefix
+
+    /// Returns a Boolean value indicating whether two values are equal.
+    ///
+    /// Equality is the inverse of inequality. For any values `a` and `b`,
+    /// `a == b` implies that `a != b` is `false`.
+    ///
+    /// - Parameters:
+    ///   - lhs: A value to compare.
+    ///   - rhs: Another value to compare.
+    @inlinable static func == (lhs: Optional, rhs: Wrapped.RawValue) -> Bool {
```

Comment:
I think that this could be removed, we probably just need the optional overload `static func == (lhs: Optional, rhs: Wrapped.RawValue?) -> Bool` :)

Reply:
@guykogus What you think about this? I really think we don't need this :)